test-case/check-smart-amplifier.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2020 Intel Corporation. All rights reserved.
+
+##
+## Case Name: check-smart-amplifier
+## Preconditions:
+##    require python3-numpy, python3-scipy and python3-matplotlib
+##    to be installed, or wavetool.py will not work
+## Description:
+##    test smart amplifier with customized wavetool.py,
+##    this tool will do binary comparison of reference wave file
+##    and recorded wave file.
+## Case step:
+##    1. acquire playback & capture pipelines with smart_amp component
+##    2. generate playback wave file (1000Hz sine wave)
+##    3. play reference wave file through smart_amp playback pipeline
+##       and record it back through smart_amp capture pipeline
+##    4. compare channel 0/1, 2/3 of recorded wave with reference wave
+## Expect result:
+##    1. reference wave and recorded wave are binary same
+##    2. delay in recorded wave less than 5ms
+##
+
+set -e
+# source from the relative path of current folder
+# shellcheck source=case-lib/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
+
+# we do not want the $TPLG to expand
```

Comment:
... because `eval` will expand it later? If correct add to comment (or explain other reason)

Reply:
this explain is ambiguous,  here we just want the string "$TPLG",  fixed